i18n/zh-CN/docusaurus-plugin-content-docs/current/design_and_concept/the_format_in_inlong.md

```diff
@@ -0,0 +1,106 @@
+---
+title: Format
+sidebar_position: 7
+---
+
+## 什么是format?
+
+![](img/format_and_flink.png)
+
+如上图所示：
+
+Flink SQL在读写数据是，均采用Row的形式，其内部为Object数组`Object[]`, 数字中每个元素代表了一个Flink表的字段。
```

Comment:
Please add one blank between the Chinese character and English character, thanks.